ext/groonga/rb-grn-database.c

```diff
@@ -623,6 +624,36 @@ rb_grn_database_unmap (VALUE self)
     return Qnil;
 }
 
+/*
+ * Recreates all index columns.
+ *
+ * This method is useful when your index column is broken.
```

Comment:
The description has less information. It should include "when you should use this method". How about the following?

This method is useful when you have any broken index columns in the database. You don't need to specify each index column. But this method spends more time rather than you specify only reindex needed index columns.